web/src/components/newsList.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import loadable from '@loadable/component';
+import { slidesToShowPlugin, arrowsPlugin } from '@brainhubeu/react-carousel';
+
+import NewsCard from './newsCard';
+import MoreButton from './moreButton';
+
+const Carousel = loadable(() => import('./carouselWrapper'));
+
+export default function NewsList({ news }) {
+    const content = {
+        text: 'More ❯',
+        back: '❮ Back',
+    };
+    return (
+        <div className="bg-brand-1 h-full mb-48 pb-48">
+            <h1 className="pt-12 text-center text-white text-4xl uppercase">
+                News
+                <span className="absolute bg-brand-4 h-1 left-[48%] text-brand-4 top-[119%] w-10" />
+            </h1>
+            <div className="mt-32 relative">
+                <Carousel
+                    plugins={[
+                        {
+                            resolve: arrowsPlugin,
+                            options: {
+                                arrowLeft: (
+                                    <MoreButton isLeft text={content.back} />
+                                ),
+                                arrowLeftDisabled: (
+                                    <MoreButton
+                                        text={content.back}
+                                        isLeft
+                                        isDisabled
+                                    />
+                                ),
```

Comment:
I know it might seem like a strange suggestion, but I don't think we need to store these text values in an object. I think passing a string is fine here.